[Screencap] Add VR180 screenshot mode

tocki1234RRR opened the pull request:
Add VR180 screenshot mode while preserving the original 360 functionality.

This simply cuts the 360 screenshot in half and adjusts the center angles. 
It also applies the camera rotation.

Original code from @mellona0821 (https://github.com/IllusionMods/BepisPlugins/issues/213).

rnetiks reviewed (commented):
Some minor formatting issues  and incorrect statements about boolean usage when type conversion exists, but that's such a minor thing I can understand why the need for an integer instead.
Having the formatting fixed would be nice.

tocki1234RRR commented:
I fixed the formatting and removed comments that might be confusing.  
I’m not very familiar with C# or English, so I may have missed some issues with formatting or comments.
Please let me know if anything still looks off. Sorry about that!

rnetiks reviewed (approved):
Thanks for the quick changes, formatting looks to be in order now and the comments are easy to grasp. I'll give my approval for this one.
